components/cdc/tests/mod.rs

```diff
@@ -130,7 +130,7 @@ fn create_event_feed(
 }
 
 pub struct TestSuiteBuilder {
-    cluster: Option<Cluster<RocksEngine, ServerCluster<RocksEngine>>>,
+    cluster: Option<Cluster<HybridEngineImpl, ServerCluster<HybridEngineImpl>>>,
```

Comment:
We should not use HybridEngine to replace RocksDB in test code. Because in real prod code, it uses RocksEngine directly. 
This is definitely not the best practice.

Reply:
Instead, we may change TestSuiteBuilder to support both engine type and then use raftstore-v2's same approach to enable the two engines with one set of test cases.